app/scripts/controllers/networks-order.tsx

```diff
@@ -0,0 +1,137 @@
+import {
+  BaseControllerV2,
+  RestrictedControllerMessenger,
+} from '@metamask/base-controller';
+import {
+  NetworkControllerStateChangeEvent,
+  NetworkState,
+} from '@metamask/network-controller';
+import type { Patch } from 'immer';
+import { MAINNET_CHAINS } from '../../../shared/constants/network';
+
+// Unique name for the controller
+const controllerName = 'NetworksOrderController';
+
+/**
+ * The network ID of a network.
+ */
+export type NetworkId = string;
+
+// State shape for NetworksOrderController
+export type NetworksOrderControllerState = {
+  networksList: NetworkId[]; // Remove the union with null
```

Comment:
Hmm, was this comment left over from something? Not sure what this is referring to

Reply:
Removed